unity/PingPongGameExample/PingPongGame/Assets/Scripts/Integration/PlatformIntegration/PlatformIntegrationUtil.cs

```diff
@@ -0,0 +1,224 @@
+﻿/**
+ * Copyright 2019 MobiledgeX, Inc. All rights and licenses reserved.
+ * MobiledgeX, Inc. 156 2nd Street #408, San Francisco, CA 94105
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using UnityEngine;
+
+// We need this one for importing our IOS functions
```

Comment:
Should clearly mark that this is not for general use. This appears best only for when the expected class fields/methods not existing is "OK" for the calling code, if merely checking the API level at runtime is insufficient in Unity.